### PR TITLE
Switched the default verifier from Degensac to Ransac

### PR DESCRIPTION
**Pull request by womackj1**
…ration files. Removed pydegensac build instructions from README.

### CODE REVIEW

**Comment by ayushbaid** [comment]
The change looks good. 

Can you present the metrics from the checks for this change, as well as from the code which has degensac on this PR.

For your PR, you can find the metrics here: https://github.com/borglab/gtsfm/runs/2336807639?check_suite_focus=true. Select the dropdown for "Run pip install pytest-cov".

As a proxy for Degensac code, use this: "https://github.com/borglab/gtsfm/runs/2336689759?check_suite_focus=true"

**Review by ayushbaid — COMMENTED** [review]
Looks good. Need to see the metrics before merging

**Comment by ayushbaid** [comment]
I am adding the useful lines from master:

```
[2021-04-13 19:28:18,547 DEBUG scene_optimizer.py line 388 3713] [Two view optimizer] [Summary] Rotation success: 66/66/66
[2021-04-13 19:28:18,547 DEBUG scene_optimizer.py line 392 3713] [Two view optimizer] [Summary] Translation success: 66/66/66
[2021-04-13 19:28:18,547 DEBUG scene_optimizer.py line 399 3713] [Two view optimizer] [Summary] Pose success: 66/66/66
[2021-04-13 19:28:18,548 DEBUG scene_optimizer.py line 403 3713] [Two view optimizer] [Summary] Image pairs with 100% inlier ratio:: 0/66


[2021-04-13 19:28:40,728 INFO bundle_adjustment.py line 108 3714] initial error: 1795566.83
[2021-04-13 19:28:40,731 INFO bundle_adjustment.py line 109 3714] final error: 59279.88
[2021-04-13 19:28:50,413 INFO bundle_adjustment.py line 114 3714] [Result] Number of tracks before filtering 9169
[2021-04-13 19:28:56,500 INFO bundle_adjustment.py line 119 3714] [Result] Number of tracks after filtering: 9119
[2021-04-13 19:28:56,509 INFO bundle_adjustment.py line 121 3714] [Result] Mean track length 5.248
[2021-04-13 19:28:56,510 INFO bundle_adjustment.py line 122 3714] [Result] Median track length 4.000
[2021-04-13 19:29:01,011 INFO gtsfm_data.py line 277 3714] Min scene reproj error: 0.000
[2021-04-13 19:29:01,012 INFO gtsfm_data.py line 278 3714] Avg scene reproj error: 0.202
[2021-04-13 19:29:01,022 INFO gtsfm_data.py line 279 3714] Median scene reproj error: 0.159
[2021-04-13 19:29:01,023 INFO gtsfm_data.py line 280 3714] Max scene reproj error: 2.975
```

File: README.md [hunk 1]
```diff
@@ -31,21 +31,14 @@ Download the Python 3.8 gtsam wheel for Mac [here](https://github.com/borglab/gt
 ```bash
 pip install ~/Downloads/gtsam-4.1.1-py3-none-any.whl
 ```
-Now, inside the environment, build pydegensac:
```
Comment (johnwlambert) [inline README.md:34]: thanks for the PR @womackj1 . Do you mind adding these instead at the end of the readme, maybe as "Compiling Additional Verifiers"?

**Comment by johnwlambert** [comment]
@ayushbaid we lose 300-400 tracks, and avg point reproj error increases by 50% from 0.20 to 0.316. But we can merge for now, maybe we can put Degensac in GTSAM later

Degensac
```
9099 tracks
Number of tracks before filtering 9099
Number of tracks after filtering: 9046
Mean track length 5.238
Median track length 4.000
 Min scene reproj error: 0.000
 Avg scene reproj error: 0.200
 Median scene reproj error: 0.158
 Max scene reproj error: 2.934
    "rotation_averaging_angle_deg": {
        "median_error": 0.09,
        "min_error": 0.02,
        "max_error": 0.19,

    "translation_averaging_distance": {
        "median_error": 0.12,
        "min_error": 0.02,
        "max_error": 0.47,

    "translation_to_direction_angle_deg": {
        "median_error": 0.35,
        "min_error": 0.04,
        "max_error": 2.23,
```


 RANSAC
```
 Number of tracks before filtering 8731
 Number of tracks after filtering: 8610
 Mean track length 5.260
 Median track length 4.000
 Min scene reproj error: 0.000
 Avg scene reproj error: 0.316
 Median scene reproj error: 0.239
 Max scene reproj error: 2.986
    "rotation_averaging_angle_deg": {
        "median_error": 0.1,
        "min_error": 0.04,
        "max_error": 0.33,

    "translation_averaging_distance": {
        "median_error": 0.05,
        "min_error": 0.02,
        "max_error": 0.16,

    },
    "translation_to_direction_angle_deg": {
        "median_error": 0.49,
        "min_error": 0.03,
        "max_error": 2.03,
         
```

File: README.md (hunk 1 above)
Comment (womackj1) [reply README.md:34]: Of course! The readme has the new section "Compiling Additional Verifiers".

**Pull request merged**